packages/react-accordion/src/components/Accordion/useAccordionStyles.ts

```diff
@@ -0,0 +1,16 @@
+import { makeStylesCompat, ax } from '@fluentui/react-make-styles';
+import { AccordionState } from './Accordion.types';
+
+/**
+ * Styles for the root slot
+ */
+const useRootStyles = makeStylesCompat<AccordionState>([]);
+
+/** Applies style classnames to slots */
+export const useAccordionStyles = (state: AccordionState) => {
+  const rootClassName = useRootStyles(state);
```

Comment:
Do we need this noop call?

Reply:
IMO no, just came by default with the scaffolding. I'll remove it!